subsys/bluetooth/controller/ll_sw/ll.c

```diff
@@ -11,7 +11,7 @@
 #include <soc.h>
 #include <device.h>
 #include <drivers/clock_control.h>
-#ifdef CONFIG_CLOCK_CONTROL_NRF
+#if defined(CONFIG_CLOCK_CONTROL_NRF) || defined(CONFIG_CLOCK_CONTROL_MPSL)
```

Comment:
I don't think we can get change to upstream zephyr. We should also try to prevent changes to zephyr, as this will take more time to get through. Do we need this change, or can we reuse CONFIG_CLOCK_CONTROL_NRF?

Reply:
Would it be ok to just do it to the fw-nrfconnect-zephyr? I was not able to make it work with CONFIG_CLOCK_CONTROL_NRF. I can try again though if this is a problem.

Reply:
I think we would try to avoid that if possible. Otherwise the two repos will diverge, which will increase maintenance cost

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/20134